packages/location-field/src/types.ts

```diff
@@ -155,7 +155,7 @@ export interface LocationFieldProps {
   locationType: LocationType;
   /**
    * A list of stopIds of the stops that should be shown as being nearby. These
-   * must be referencable in the stopsIndex prop.
+   * must be referenceable in the stopsIndex prop.
```

Comment:
codespell has once again woken from its slumber to find spelling problems in completely separate parts of the code from the one i edited